### PR TITLE
Added hook

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
fixes:

### CODE REVIEW

File: demisto_sdk/commands/pre_commit/.pre-commit-config_template.yaml [hunk 1]
```diff
@@ -27,26 +27,33 @@ repos:
...
+      #     [
+      #       --ignore-missing-imports,
+      #       --check-untyped-defs,
+      #       --show-error-codes,
+      #       --follow-imports=silent,
+      #       --allow-redefinition,
+      #       # this will be replaced by the python version in the runtime
+      #       --python-version=3.10,
+      #     ]
+      #   exclude: "test_data|tests_data|.venv|.*_test.py$|infrastructure_tests"
```
Comment (dorschw) [inline demisto_sdk/commands/pre_commit/.pre-commit-config_template.yaml:55]: intended?

File: demisto_sdk/commands/pre_commit/hooks/hook.py [hunk 2]
```diff
@@ -8,3 +11,10 @@ def __init__(self, hook: dict) -> None:
     @abstractmethod
     def prepare_hook(self, **kwargs):
         return self.hook
+
+    def additional_dependencies(self, group_name: str) -> List[str]:
+        factory = Factory()
+        poetry = factory.create_poetry(CONTENT_PATH)
+        pkg = poetry.package
+
+        return [p.to_pep_508() for p in pkg.dependency_group(group_name).dependencies]
```
Comment (dorschw) [inline demisto_sdk/commands/pre_commit/hooks/hook.py:20]:
```suggestion
        dependencies = Factory().create_poetry(CONTENT_PATH).package(group_name).dependencies
        return [p.to_pep_508() for p in dependencies]
```

**Comment by GuyAfik** [comment]
@shmuel44 @dorschw is this PR still relevant?

**Comment by shmuel44** [comment]
> @shmuel44 @dorschw is this PR still relevant?

@ilaner ?

**Pull request closed without merging**